doc/getting-started.rst

```diff
@@ -0,0 +1,120 @@
+Getting Started
+===============
+
+Dependencies
+************
+
+* Python >= 3.2
+* Numpy
+* Sklearn
+* Scipy
+
+Installation
+************
+Open-SIR and all its dependencies can be installed from the repository using
+`pip`:
+::
+
+    git clone https://github.com/open-sir/open-sir.git
+    cd open-sir
+    pip install .
+
+In order to unintall Open-SIR simply execute:
+::
+
+    pip uninstall opensir
+
+Usage example
+*************
+
+You can use Open-SIR to create a 6 days prediction of the number of susceptible (S), infected (I) and removed (R) population. 
+The initial conditions `-i` represent Ealing data as of 04/04/2020. The parameters `-p` provide a prediction in the hypothetical 
+case that no lockdown would be taking place.
```

Comment:
Just curious, how much chars per line are you using?

Reply:
it should be 80, but I forgot to apply formats here. I can adapt it :)